### PR TITLE
Newsweek milestone2

### PR DESCRIPTION
**Pull request by prolajumokeoni**
# Newsweek-Milestone2
In this milestone:
> We created the three columns sections of the main section
> We  made the columns responsive on mobile, medium and desktop using bootstrap classes
> We re-arranged the columns to ensure they move to the correct location on mobile and medium
#### We have implemented all changes requested in the previous review.

### CODE REVIEW

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
# Status: Change Requested

Wow, good job guys @prolajumokeoni and @blakbox23 your project looks good so far just a few things you have to work on, and be ready to move on

## Highlights image

- PR is very professional
- Your project looks good on a mobile device
- It looks good on the Desktop
- Used semantic HTML
- Commit messages is descriptive

## To Improve image

### Tablet

original

down

yours

down1

- Kindly remove an empty space on top of the navbar
sp
- On the desktop remember there is no hamburger icon that needs to appear
- Every image is supposed to have a link
- Pay attention to details like

details

## Please refer to the original while designing your project and work on the requested change and submit another CRR and remember you only have 2 CR left and work accordingly 

# Happy Coding

Lydia Ingabire

File: index.html [hunk 1]
```diff
@@ -0,0 +1,764 @@
...
+        </div>
+
+        <div class="position-relative">
+          <div class="versus position-absolute rounded-circle bg-dark">
+            <p class="px-3 mt-2 text-muted fw-bolder">vs</p>
+          </div>
+          <div class="border-bottom border-5">
+            <div class="border-top mt-1 d-flex align-items-center justify-content-between pt-4">
+
+              <img class="p-3 order-2" src="https://web.archive.org/web/20210120125445im_/https://d.newsweek.com/en/full/1673623/froma-harrop.png?w=63&h=63&f=2ba48b46c4bcf031c010ff486c04d55b" alt="froma-harrop">
```
Comment (descholar-ceo) [inline index.html]: [required] As it was mentioned in the previous review, [all images should have a link] => so does this image: it should have a link

**Review by descholar-ceo — CHANGES_REQUESTED** [review]
# Status: Changes Required :vertical_traffic_light: :warning: 
## Hello @prolajumokeoni and @blakbox23  :handshake: :handshake: :handshake: 

:+1: This is good to make it this far, your clone almost looks like the original page and it is really nice, congratulations on the good work you have done so far.
:+1: I liked that you have tried to implement the requested changes from the previous review, keep it up, and your project is really almost done.
#### :warning: However we need to make few changes to mark this project approved, don't feel discouraged because of being asked some changes more and more, that's because this is a learning process, and we are here to help you to become an out-standing developer. See the requested change below :point_down: :point_down: :point_down: 
### Note: All of requested changes which are marked as [Required] are mandatory to be implemented in order to get this project approved, others which are marked as [Optional] are nice to have in your project to make it outstanding

# [Required Changes] :red_circle: 
- [x] First change: please read my inline comment: `Every image is supposed to have a link`
- [x] Kindly remove an empty space on top of the navbar

| Your clone | Original page |
| --- | --- |
| Screenshot from 2021-04-07 12-16-00| Screenshot from 2021-04-07 12-17-01|

# [Optional changes] :yellow_circle: 
- [ ] Navbar is messed on mobile screens of width 320px which is not the case on the original website
- [ ] I would advise you to implement the polygonal right end of the navbar for the learning purposes
___

_:warning: Reminder: As described in the [Code reviews limit Policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561#), when you reach to the limit of 3 code reviews and your project was not approved, you will need to repeat the project. => You are now left with only one (1) chance to get a review on this project. So implement the requested changes very carefully. If you think that the code review was not fair, you can request a second-opinion using [this form](https://airtable.com/shrmBIv6cXYrq4Fgv)._

_If anything seems to be unclear to you, reach out and ask anything by commenting on this Pull Request and mention me as [descholar-ceo](https://github.com/descholar-ceo), I will receive a notification and will respond._

[GitHub](https://github.com/descholar-ceo) | [LinkedIn](https://linkedin.com/in/mugirase-emmanuel) | [Twitter](https://twitter.com/descholar3)

_Reviewed by [Mugirase Emmanuel (descholar)](https://github.com/descholar-ceo)_

File: index.html (hunk 1 above)
Comment (blakbox23) [reply index.html]: We've now included the a tag for this image. @descholar-ceo

**Review by MkrtichSargsyan — APPROVED** [review]
## _Status:_ Approved  :hamster: :rocket:

## Hi Guys.

Good job on implementing the changes requested, your project is now approved, 


## OPTIONAL

- [ ] Add h1 tag to the page

  theoretically and logically wrong to have subheadlines ```<h2>``` without a main headline ```<h1>```,   but it won't have a negative effect.


You can go ahead and merge this branch to the base branch of this repo.

Good luck in your future projects and happy coding :relaxed:

> ps: . If you have any questions about my review or need help, you can message me on slack ( @mkrtichsargsyan )

**Comment by blakbox23** [comment]
@MkrtichSargsyan Thank you for the review. We plan to handle all optional changes after this milestone. 
Thanks

**Comment by prolajumokeoni** [comment]
@MkrtichSargsyan we would do the optional changes, thanks!

**Pull request closed without merging**